app/src/main/java/fr/free/nrw/commons/Media.java

```diff
@@ -486,11 +485,7 @@ public static Media from(MwQueryPage page) {
                 StringUtils.getParsedStringFromHtml(metadata.artist().value())
         );
 
-        String language = Locale.getDefault().getLanguage();
-        if (StringUtils.isNullOrWhiteSpace(language)) {
-            language = "default";
-        }
```

Comment:
@maskaravivek can you explain why this lines were unnecessary?

Reply:
Thanks for pointing it out. I guess it got removed while merging with the master. Will add it again.